src/libraries/System.Threading.Tasks.Dataflow/tests/Dataflow/DataflowTestHelper.IAsyncEnumerable.cs

```diff
@@ -0,0 +1,42 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Collections.Generic;
+
+namespace System.Threading.Tasks.Dataflow.Tests
+{
+    internal static partial class DataflowTestHelpers
+    {
+        internal static Func<int, IAsyncEnumerable<int>> ToAsyncEnumerable = item => AsyncEnumerable.Repeat(item, 1);
+    }
+
+    internal static partial class AsyncEnumerable
+    {
+#pragma warning disable 1998
```

Comment:
CS1998 is "just" that we use async here without an awaiting expression (imo this should be suppressed project-wide)

```suggestion
// The Diagnostic CS1998 has been explicitly suppressed here, because it is a public API that exposes an IAsyncEnumerable without awaiting.
#pragma warning disable 1998
```